### PR TITLE
Error fixes 2 testing and documentation

### LINKED ISSUE #84: [Error Fixes 2] Testing and documentation

### PR DESCRIPTION
**Pull request by stephaniesaab**
# Description

Finished testing coverage for the entities folder. 100% Coverage of the methods and lines obtained. Also removed some redundant methods that were only used in the tests (e.g. getters and setters that were only used to test entity objects).

Fixes #84  (Error 2)

## Type of change

- [ ] Compilation error fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation
- [X] Test cases
- [X] Redundant code removal (fix code smells)
- [ ] Data storage 
- [ ] UI element

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/useCases/ModifyTaskUC.java [hunk 1]
```diff
@@ -39,8 +39,7 @@ private boolean findTask(String taskName, User user) {
     public boolean findTask (String taskName) {
         return findTask(taskName, UserUC.u());
     }
-
-    /**
```
Comment (harshsbajwa) [inline src/main/java/useCases/ModifyTaskUC.java:43]: you uncommented something, please change

**Review by harshsbajwa — CHANGES_REQUESTED** [review]
great work, excellent documentation, please fix the comment in modifytaskuc

**Review by XinyingXu — APPROVED** [review]
Good job on the tests, documentation and error fixes.

**Review by serenahonekin — APPROVED** [review]
Good work on adding test cases and documentation!

**Review by e-laflamme — APPROVED** [review]
Looks good!

**Pull request merged**